src/common/gui/base/Toggle.ts

```diff
@@ -0,0 +1,98 @@
+import m, { ClassComponent, Vnode } from "mithril"
+import { Keys, TabIndex } from "../../api/common/TutanotaConstants.js"
+import { isKeyPressed } from "../../misc/KeyManager.js"
+
+type ToggleVariant = "normal" | "expanded"
```

Comment:
Does an expanded button take up the whole line?

Reply:
Yes